src/core/server/config.ts

```diff
@@ -377,6 +389,12 @@ const config = convict({
     default: ms("1m"),
     env: "NODEJS_HEADERS_TIMEOUT",
   },
+  amp_cache_domains: {
+    doc: "Specifies the amp cache domains as a comma seperated list",
+    format: "list",
+    default: ["cdn.ampproject.org", "www.bing-amp.com"],
```

Comment:
I really think we don't need to add `www.bing-amp.com` it looks like their format used to be `https://www.bing-amp.com/c/s/yourdomain.com/yourpath`, and it seems that it's not being used anymore. Have you found any evidence that this still exists?